classes/class-rest-mappings-controller.php

```diff
@@ -225,16 +267,7 @@ public function prepare_item_for_response( $item, $request ) {
 			return new WP_REST_Response( $item );
 		}
 
-		$data = array(
-			'id'     => absint( $item->get_id() ),
-			'domain' => $item->get_domain(),
-			'active' => $item->is_active(),
-		);
-
-		// Return blog ID if sent
-		if ( null !== $request->get_param( 'blog' ) ) {
-			$data['blog'] = $request->get_param( 'blog' );
-		}
+		$data = $this->mapping_to_array( $item, $request );
```

Comment:
I realised I was preparing the response for the `get_items()` call too, but was returning an array of responses from that - just needed to be able to map a group of items to an array as well as an individual one before making that data into a response.